app/services/allocation_services/create.rb

```diff
@@ -0,0 +1,36 @@
+module AllocationServices
+  class Create
+    include PublishService
```

Comment:
Sorry to nitpick about naming, but I thought this service had something to do with publishing a course, whereas having read the module below I see it's probably about exposing `call` as a class method. Would it make sense to use a name that's more orthogonal to this service?

Reply:
Fair point. I named it meaning 'a service in the Publish app' but I see the confusion.